test/unix.js

```diff
@@ -19,6 +20,12 @@ if (process.platform !== 'win32') {
     })
     t.teardown(server.close.bind(server))
 
+    try {
+      fs.unlinkSync('/var/tmp/test3.sock')
+    } catch (err) {
+
```

Comment:
why is this empty?

Reply:
This is just in case the test crashed in earlier run and there is a orphaned file.